### ISSUE TITLE
coreutils:  Missing dependency "crypto"

### ISSUE THREAD

**Issue by GordonSmith** [phosphorjs/phosphor#353]
random.ts has a "require" for "crypto", but package.json has no reference to that package:
https://github.com/phosphorjs/phosphor/blob/51a653011dae5533dcc3097e193b70b32ff9fe24/packages/coreutils/src/random.ts#L42

https://github.com/phosphorjs/phosphor/blob/master/packages/coreutils/package.json

This is apparently causing issues downstream in Angular 6.

**Comment by blink1073** [comment]
Hi @GordonSmith, see discussion in https://github.com/phosphorjs/phosphor/pull/349.

**Comment by GordonSmith** [comment]
@blink1073 That PR works 100% for me (and all browser users) - but if there is a genuine need to fallback to the Node crypto library for some reason (the coreutils might be used in other scenarios like unit tests etc.), then my proposed solution might be more acceptable to @sccolbert

**Comment by blink1073** [comment]
Does `webpack` honor the `browser` field for the entry point?

**Comment by GordonSmith** [comment]
Yes (99% sure) - `webpack` defaults to `browser = true` as well (which shouldn't be a surprise).

**Comment by SimonBiggs** [comment]
@gordonsmith my method was a hacky workaround to be sure. We should close my pull request and replace it with yours.
